### PR TITLE
Small modifs to make code run

### PR DESCRIPTION
**Pull request by manhbao-nguyen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@ RLAptamer/
 sequenceModel2
 distanceTests.py
 sh/
+
+*.txt
```
Comment (manhbao-nguyen) [inline .gitignore:13]: record.txt was otherwise git pushed

File: main.py [hunk 2]
```diff
@@ -476,6 +476,18 @@ def add_args(parser):
...
+        "--reward_func",
+        default="power",
+        type=str,
+        help="Function for rewards transformation: power or boltzmann",
+    )
+    args2config.update({"reward_func": ["gflownet", "reward_func"]})
+
+    parser.add_argument("--temperature_logits", default=1.0, type=float)
+    args2config.update({"temperature_logits": ["gflownet", "temperature_logits"]})
+
```
Comment (manhbao-nguyen) [inline main.py:490]: params missing in the main.py / present in gflownet.py only.
With hydra we will change this anyway ;)

File: main.py [hunk 3]
```diff
@@ -642,9 +654,9 @@ def process_config(config):
     config.gflownet.func = config.dataset.oracle
     config.gflownet.test.score = config.gflownet.func.replace("nupack ", "")
     # Comet: same project for AL and GFlowNet
+    config.al.comet.project = config.comet_project
```
Comment (manhbao-nguyen) [inline main.py:657]: we have to set config.al.comet.project anyway

File: models.py [hunk 4]
```diff
@@ -189,8 +189,7 @@ def getLoss(self, train_data):
 
     def getMinF(self):
         inputs = self.dataset_samples
-        if self.config.device == 'cuda':
-            inputs = torch.Tensor(inputs).cuda()
+        inputs = torch.Tensor(inputs).to(self.config.device)
```
Comment (manhbao-nguyen) [inline models.py:192]: AttributeError: 'numpy.ndarray' object has no attribute 'long' otherwise

File: models.py [hunk 5]
```diff
@@ -308,6 +307,8 @@ def raw(self, Data, output="Average"):
                 return np.var(out,axis=1)
             elif output == 'Both':
                 return np.average(out,axis=1), np.var(out,axis=1)
+            else:
+                return np.average(out, axis = 1), np.var(out, axis = 1)
```
Comment (manhbao-nguyen) [inline models.py:311]: in runSampling in querier.py, batch2dict is called with query_function=self.config.al.query_mode = 'learned' by default, which is not taken into account by current raw in models.py

**Review by alexhernandezgarcia — APPROVED** [review]

**Pull request merged**